angular-workspace/projects/ni/nimble-angular/src/test.ts

```diff
@@ -24,3 +24,6 @@ getTestBed().initTestEnvironment(
 const context = require.context('./', true, /\.spec\.ts$/);
 // And load the modules.
 context.keys().map(context);
+
+// eslint-disable-next-line no-console, @typescript-eslint/no-explicit-any
+console.error = (data: any): void => fail(data);
```

Comment:
Can we add the link to where we are pulling this workaround from to both spots, ie https://github.com/angular/angular/issues/36430#issuecomment-874772398

Reply:
Also looks like this is a temporary workaround we can remove in Angular 14, can we capture that somewhere so we can remove it and switch to the angular supported approach on upgrade? https://github.com/angular/angular/issues/36430#issuecomment-1117801535

Reply:
Added links and referenced this PR and that Angular issue from #270